packages/dds/merge-tree/src/test/mergeTreeOperationRunner.ts

```diff
@@ -29,6 +29,59 @@ export type TestOperation = (
 	random: IRandom,
 ) => IMergeTreeOp | undefined;
 
+export const insertField: TestOperation = (
+	client: TestClient,
+	opStart: number,
+	opEnd: number,
+	random: IRandom,
+) => {
+	const numberText = random.string(5, "0123456789");
```

Comment:
maybe do something like `(client.longClientId.codePointAt(0) %10).repeat(5)` so that each client gets a uniqe-ish field id which can make debugging and reading logs easier.

Reply:
It would also be good to have various lengths.

Reply:
It looks like the subsequent code prevents adjacency.